app/controllers/photos_controller.rb

```diff
@@ -1,39 +1,31 @@
 class PhotosController < ApplicationController
   def index
     @photos = current_user.photos(&:images)
-   end
-  
-   def new
+  end
+
+  def user_pictures
+    @user = User.find(params[:format])
+    @photos = @user.photos(&:images)
+  end
+   
+  def new
     @photo = Photo.new
-   end
+  end
   
-   def create
+  def create
     photos = Photo.new
     
     photos.images << photos.images.attach(params[:photo][:image])
+    photos.name =params["title"]["{:class=>%22form-control%22}"]
```

Comment:
I have no idea what you intend doing with this line. I believe it can be much better. Have you heard of strong params? Can you rather use that here to clean this up?